src/gridfs/download.ts

```diff
@@ -196,7 +204,8 @@ export class GridFSBucketReadStream extends Readable {
   async abort(): Promise<void> {
     this.push(null);
     this.destroy();
-    await this.s.cursor?.close();
+    const remainingTimeMS = getRemainingTimeMSOrThrow(this.s.timeoutContext);
+    await this.s.cursor?.close({ timeoutMS: remainingTimeMS });
```

Comment:
The spec says that abort should fall under the overall lifetime of the stream, but wouldn't this leak server resources if it were to time out before the command is written?

Reply:
Is this question "new"? in the sense that it is the same one we asked about timing out iterations on the cursor right?

Reply:
Okay yes worth calling out but something we should defer and track the answer to. I think the prevailing feeling is that we rely on auto clean up server-side but we should document the answer when it is clear to us and users what the expectation is.